### DISCUSSION TITLE
在onFormFirstQuery中间件中，不调用next，设置默认不查询失效

### DISCUSSION THREAD

**Discussion by connie-KK** [alibaba/formily#1846] [Q&A 1.x] [answered]
在onFormFirstQuery中间件中，不调用next，设置默认不查询，请求没有发起，但是返回的table中的loading状态没有更新，依然为true，复现地址：https://codesandbox.io/s/cancelfirstquerymiddleware-forked-6w2kl?file=/index.js

**Comment by janryWang** [comment, marked as answer]
目前中间件机制是有个bug，没法阻塞后续流程，你想要阻塞的话，可以return new Promise()，不resolve来阻塞

**Reply by connie-KK** [reply]
可以帮忙提供个demo吗～
